src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -368,89 +370,117 @@ ErrorIfQueryNotSupported(Query *queryTree)
 	bool hasComplexJoinOrder = false;
 	bool hasComplexRangeTableType = false;
 	bool preconditionsSatisfied = true;
+	char *errorHint = NULL;
+	char *joinHint = "Consider joining tables on partition column and have equal "
```

Comment:
Is the sentence trying to say `consider('joining on partition column' AND 'adding an equality filter')` or is it saying `consider('joining on partition column') AND consider('adding an equality filter')`? I can only see a case for the latter, syntax-wise.

Maybe: _Try joining the tables on their partition column or include an equality filter on the join column_? Or does that mean something different?